applications/API/models.py

```
@@ -0,0 +1,733 @@
+            instance.set_insult()  # Ensure the related Insult is set
+
+        # Use atomic update with F() expression for performance
+        if instance.insult_id:
+            Insult.objects.filter(pk=instance.insult_id).update(
+                status=Insult.STATUS.FLAGGED,
+                reports_count=F('reports_count') + 1
+            )
+
+
+@receiver(post_delete, sender=InsultReview)
+def decrement_report_count(sender, instance, **kwargs):
```

Ruff (0.14.1)

Unused function argument: `sender`

(ARG001)